### DISCUSSION TITLE
Connection Pooling -  Lambda Context

### DISCUSSION THREAD

**Discussion by mousedown** [vercel/vercel#4548] [General] [answered]
MongoDB connection sharing [doesn't seam possible](https://github.com/vercel/next.js/discussions/12229) on Now using Next. I suspect what's needed is to configure Lambda’s context with `context.callbackWaitsForEmptyEventLoop`

Can the api handler with context be accessed in the same way you would via serverless? 

`exports.handler = (event, context, callback) => {}`

**Comment by styfle** [comment, marked as answer]
We automatically assign `callbackWaitsForEmptyEventLoop` to false. No action needed on your part.

https://github.com/vercel/vercel/blob/45d05a603b0d883461e5ed6aa5d1bf18d4a3cc55/packages/now-node-bridge/src/bridge.ts#L180

**Comment by mousedown** [comment]
Thank you
